test/visualization/test_merge_masks.py

```diff
@@ -16,16 +16,16 @@ def setup(self):
         self.fullPath = Path(__file__).resolve().parent
         # Move up to the test directory, "test/"
         self.testPath = self.fullPath.parent
-
+        self.output_filename = 'axon_myelin_mask.png'
```

Comment:
Just curious, why not using the standard axonmyelin suffix here?

Reply:
Resolved in b9d9eb9